QC.Sty/pyophidia/find_oph_workflows.py

```diff
@@ -0,0 +1,59 @@
+from PyOphidia import client
+import os
+import json
+import argparse
+
+
+def find(pattern, path):
+    result = []
+    for root, dirs, files in os.walk(path):
+        for name in files:
+            if pattern in name:
+                result.append(os.path.join(root, name))
+    return result
+
+
+def get_input_args():
+    parser = argparse.ArgumentParser(description=("Find Ophidia workflows"))
+    parser.add_argument(
+        "--path", metavar="PATH", type=str, help="path to look for in the repository"
```

Comment:
The type of argument has to match (and viceversa) with the definition in tooling. Right now, the script expects an optional argument `--path` such as in:

`QC.Sty/pyophidia/find_oph_workflows.py --path .`

and the tool definition (in tooling) has a positional argument, so currently expects:

`QC.Sty/pyophidia/find_oph_workflows.py .`